README.md

```diff
@@ -1 +1,70 @@
-# Space-Travelers-Hub
\ No newline at end of file
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
```

Comment:
Please remove the README CRA boilerplate, add your own

Reply:
Hello! I have updated the READme.md file. 👍🏼